File: Infra/Entity/Item.cs
using System.Collections.Generic;

namespace RestauranteApi.Infra.Entity
{
  public class Item
  {
    public int Id { get; set; }
    public string Titulo { get; set; }
    public string Descricao { get; set; }
    public string UrlImagem { get; set; }
    public string Categoria { get; set; }
    public double Valor { get; set; }
    public ICollection<ItemPedido> ItemPedido { get; set; }
  }
}

File: src/infra/models/item.js
module.exports = (sequelize, DataTypes) => {
  const Item = sequelize.define('Item', {
    id: {
      autoIncrement: true,
      primaryKey: true,
      type: DataTypes.INTEGER,
      field: 'Id'
    },
    category: {
      type: DataTypes.STRING,
      field: 'Categoria'
    },
    description: {
      type: DataTypes.STRING,
      field: 'Descricao'
    },
    name: {
      type: DataTypes.STRING,
      field: 'Titulo'
    },
    imagePath: {
      type: DataTypes.STRING,
      field: 'UrlImagem'
    },
    value: {
      type: DataTypes.DECIMAL,
      field: 'Valor'
    }
  }, {
    tableName: 'Item',
    undercored: false,
    updatedAt: false,
    createdAt: false
  })
  return Item
}
